src/styles/homePage.css

```diff
@@ -0,0 +1,85 @@
+.classA {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+}
+
+.quiz-container-bg {
+	background: var(--color-bg);
+	color: var(--color-text-1);
+}
+
+.card-container-bg {
+	background: var(--color-card-bg);
+	color: var(--color-text-1);
+}
+
+.abc {
+	display: flex;
+}
+
+.ab {
```

Comment:
Try to give more meaningful names to your classes

Reply:
Okay, I'll do that.